docs/azure_arc_jumpstart/azure_arc_servers/oci/oci_terraform_ubuntu/_index.md

```diff
@@ -0,0 +1,199 @@
+ 
+   ```shell
+     $cat oci_api_key_public.pem | gvim &
+  ```
+
+The public key needs to be added to your user account in the OCI console. Open the [account page](https://cloud.oracle.com/identity/users/) for your user, navigate to "API Keys" and select the "Add Public Key" button. Copy and paste the contents of the oci_api_key_public.pem file in the box of the "Add Public Key" dialog as shown below.
+
+  ![Screenshot of OCI cloud console](./04.png)
+
+ ## Configure Terraform
+
+Before executing the Terraform plan, you must export the environment variables which will be used by the plan. These variables are based on your Azure subscription and tenant, the Azure service principal, and the OCI OCIDs and keys you just created.
+
+* Retrieve your Azure Subscription ID and tenant ID using the ```az account list``` command.
+
+* The Terraform plan creates resources in both Microsoft Azure and OCI. It then executes a script on an OCI compute instance to install the Azure Arc agent and all necessary artifacts. This script requires certain information about your OCI and Azure environments. Edit [*scripts/vars.sh*](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/oci/ubuntu/terraform/scripts/vars.sh) and update each of the variables with the appropriate values.
+
+    * TF_VAR_tenancy_ocid=Oracle tenanacy OCID
+    * TF_VAR_user_ocid=Oracle user OCID
+    * TF_VAR_fingerprint=Oracle API Key FingerPrint
+    * TF_VAR_private_key_path=oci_api_key.pem
+    * TF_VAR_ssh_public_key=$(cat my_oci_key.pub) 
+    * TF_VAR_region=us-ashburn-1
+    * TF_VAR_compartment_ocid=Oracle compartment OCID
+    * TF_VAR_subscription_id=Your Azure subscription ID
+    * TF_VAR_client_id=Your Azure service principal app id
+    * TF_VAR_client_secret=Your Azure service principal password
+    * TF_VAR_tenant_id=Your Azure tenant ID
+
+* From CLI, navigate to the "azure_arc_servers_jumpstart/oci/ubuntu/terraform"  directory of the cloned repo.    
+  
+* Export the environment variables you edited by running [*scripts/vars.sh*](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/oci/ubuntu/terraform/scripts/vars.sh) with the source command as shown below. Terraform requires these to be set for the plan to execute properly. Note that this script will also be automatically executed remotely on the OCI virtual machine instance as part of the Terraform deployment. 
+* 
+    ```shell
+    source ./scripts/vars.sh
+    ```
+
+* Run the ```terraform init``` command which will download Terraform's AzureRM and OCI providers.
+
+    ![Screenshot of terraform init being run](./05.png)
+
+# Deployment
+
+* Run the ```terraform apply --auto-approve``` command and wait for the plan to finish. Upon completion, you will have an OCI Virtual Machine Instance deployed and connected as a new Azure Arc enabled server inside a new Resource Group.
```

Comment:
```suggestion
* Run the ```terraform apply --auto-approve``` command and wait for the plan to finish. Upon completion, you will have an OCI virtual machine instance deployed and connected as a new Azure Arc enabled server inside a new Resource Group.
```

Reply:
is it compute instance or virtual machine instance? I've seen both used so far.